akka-actor-typed/src/main/scala/akka/actor/typed/internal/MiscMessageSerializer.scala

```diff
@@ -1,16 +1,15 @@
 /**
  * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
  */
-package akka.cluster.typed.internal
+package akka.actor.typed.internal
 
+import java.io.NotSerializableException
 import java.nio.charset.StandardCharsets
 
+import akka.actor.typed.{ ActorRef, ActorRefResolver }
+import akka.actor.typed.scaladsl.adapter._
 import akka.annotation.InternalApi
 import akka.serialization.{ BaseSerializer, SerializerWithStringManifest }
-import akka.actor.typed.ActorRef
-import akka.cluster.typed.ActorRefResolver
-import akka.actor.typed.scaladsl.adapter._
-import java.io.NotSerializableException
 
 @InternalApi
 class MiscMessageSerializer(val system: akka.actor.ExtendedActorSystem) extends SerializerWithStringManifest with BaseSerializer {
```

Comment:
while touching this could you extract the string manifest "a" to a constant

Reply:
done